packages/themes/src/themes/console/light.ts

```diff
@@ -4,6 +4,14 @@
  * PLEASE DO NOT EDIT HERE
  */
 export const lightTheme = {
+  breakpoints: {
```

Comment:
can we sort ?

Reply:
@philibea why would you want to sort it? 😅

Reply:
in size order :)

Reply:
@philibea Im not sure it's useful the tokens are use as object directly, ordering will need to increase the generating script size for not much I think